db/migrate/20220120110445_make_notes_user_not_nullable.rb

```diff
@@ -0,0 +1,5 @@
+class MakeNotesUserNotNullable < ActiveRecord::Migration[6.1]
+  def change
+    add_check_constraint :notes, "user_id IS NOT NULL", name: "notes_user_id_null", validate: false
```

Comment:
TIL! https://github.com/ankane/strong_migrations#good---postgres-1
Which PG version are we using, as can get away with reinstating the `null: false` in the second migration?